language/grounding/intro-grounding.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<p>This output doesn't include any Google Search sources, is that intended?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Looks like the other chat examples don't have citations either.

Reply:
Not intended. In earlier versions of the Python client lib, `WebSearch()` never returned sources and chat models were not working with the `grounding_source` kwarg at all. So some progress in the latest lib version. I'll follow up on the missing citations in a separate bug to the appropriate team.